modules/webxr/webxr_interface_js.cpp

```diff
@@ -484,7 +473,17 @@ bool WebXRInterfaceJS::pre_draw_viewport(RID p_render_target) {
 Vector<BlitToScreen> WebXRInterfaceJS::post_draw_viewport(RID p_render_target, const Rect2 &p_screen_rect) {
 	Vector<BlitToScreen> blit_to_screen;
 
-	// We don't need to do anything here.
+	GLES3::TextureStorage *texture_storage = dynamic_cast<GLES3::TextureStorage *>(RSG::texture_storage);
+	if (texture_storage == nullptr) {
+		return blit_to_screen;
+	}
+
+	GLES3::RenderTarget *rt = texture_storage->get_render_target(p_render_target);
```

Comment:
I think this would be cleaner if we have a `render_target_set_reattach_textures` here, access member variables of `RenderTarget` outside of `drivers/gles3/*` is frowned upon.

Reply:
Changed in my latest push